lambdasystemservice/src/main/java/db/modal/FunctionType.java

```diff
@@ -0,0 +1,11 @@
+package db.modal;
+
+/**
+ * Created by deshan on 9/24/17.
+ */
+public enum FunctionType {
+    JAVA,
```

Comment:
we can use this https://github.com/ThilinaManamgoda/serverless-architecture/blob/master/lambdacli/src/main/java/core/utilities/FileType.java